pom.xml

```diff
@@ -194,15 +148,8 @@
             <plugin>
                 <groupId>org.eluder.coveralls</groupId>
                 <artifactId>coveralls-maven-plugin</artifactId>
-                <version>4.2.0</version>
-            </plugin>
-            <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>findbugs-maven-plugin</artifactId>
```

Comment:
Why are you removing findbugs?

Reply:
I just removed the explicit configuration here because this is now already done in the parent pom.